packages/hooks/use-in-view.ts

```diff
@@ -55,6 +56,13 @@ const useInView: UseInView = ({ rootMargin, onlyOnce } = {}) => {
   const [isIntersecting, setIntersecting] = useState(false);
   const ref = useRef();
 
+  useEffect(() => {
+    error(
+      true,
+      "The @frontity/hooks package is deprecated.\nThe Frontity team recommends that you use https://github.com/thebuilder/react-intersection-observer instead."
+    );
```

Comment:
This has to be a deprecation warning, not an error. Also, it's not the `@frontity/hooks` package, but only the `useInView` hook.

Reply:
So, I was thinking that we could follow the facebook strategy by the way, and show all warnings as errors (like react does), but I don't have a strong opinion on that.

Reply:
Yes, that was my thought too but this deprecation message should be a warning and not an error, shouldn't it?